packages/taxdat/R/postprocessing.R

```diff
@@ -760,3 +760,29 @@ cri_interval <- function() {
 custom_quantile2 <- function(x, cri = cri_interval()) {
   posterior::quantile2(x, probs = cri)
 }
+
+
+#' get_coverage
```

Comment:
It doesn't seem like the figure where this function is called was committed.

Reply:
Yes this the figure is in a scratchpad for now.